org-journal.el

```diff
@@ -1880,6 +1879,11 @@ Only one recipient is supported.  ")))
                    'before-save-hook)
       (save-buffer))))
 
+(defcustom org-journal-mode-hook
+  '(visual-line-mode)
```

Comment:
This toggles visual-line-mode instead of turning it on. I wonder if this might lead to problems if, say, org-mode-hook contained visual-line-mode as well?

Reply:
Ah, thanks for catching that. I'll test it and change it if necessary.

Reply:
It doesn't seem to have an effect, but I've made the change anyway.